assets/app/view/game_card.rb

```diff
@@ -155,23 +156,52 @@ def render_body
       }
 
       p_elm = players.map.with_index do |player, index|
-        elm = h(
+        elms = []
+
+        elms << h(
           acting?(player) ? :u : :span,
-          { style: { 'margin-right': '0.5rem' } },
-          player['name'] + (index != (players.size - 1) ? ',' : ''),
+          player['name']
         )
 
+        elm_props = {
+          style: {
+            'margin-right': '0.5rem',
+            'white-space': 'nowrap',
+          },
+        }
+
         if owner? && new? && player['id'] != @user['id']
-          button_props = {
-            on: { click: -> { kick(@gdata, player) } },
-            style: {
-              'margin-left': '0.5rem',
-            },
-          }
-          elm = h('button.button', button_props, [elm])
+          id = "#{@gdata['id']}-#{player['id']}"
+          if @confirm_kick != id
+            button_props = {
+              on: { click: -> { store(:confirm_kick, id) } },
+              style: {
+                color: 'red',
+                cursor: 'pointer',
+              },
+            }
+
+            elms << h('a', button_props, '❌')
```

Comment:
i think this could be dryed up a bit

maybe put button_props out of the block and do

button_props = { 
  style: { cursor: 'pointer' },
  on: {},
}